packages/angular-cli/blueprints/ng2/files/__path__/tsconfig.json

```diff
@@ -12,6 +12,11 @@
     "target": "es5",
     "typeRoots": [
       "../node_modules/@types"
-    ]
+    ],
+    "baseUrl": ".",
+    "paths": {
+      "@<%= prefix %>/*": ["app/*"],
+      "app/*": ["app/*"]
```

Comment:
ok, i will change to just get the paths defined in the tsconfig.json applied to webpack resolve without defaults.